### PR TITLE
Update limits-for-content-search.md

### PR DESCRIPTION
**Pull request by shelri**
This is a part of updating 4k limit details for AED and Core Ediscovery as we have seen an influx in customer escalations related to this information.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/limits-for-content-search.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The following table lists the search limits when using the content search tool i
...
+|The maximum number of characters for the search query (including operators and conditions) for a search.  <br/><br/> **Note:** This limit takes effect after the query is expanded and includes characters from the Query, Securifty Filters applied to the user and all locations URL length. This means the query will get expanded against each of the keywords. For example, if a search query has 15 keywords and additional parameters and conditions, the query gets expanded 15 times, each with the other parameters and conditions in the query. So even though the number of characters in search query may be below the limit, it's the expanded query that may contribute to exceeding this limit.  <br/> |**Mailboxes:** 10,000  <br/> **Sites:** 4,000 when searching all sites or 2,000 when searching up to 20 sites <sup>2</sup> <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/limits-for-content-search.md]:
```suggestion
|The maximum number of characters for the search query (including operators and conditions) for a search.  <br/><br/> **Note:** This limit takes effect after the query is expanded and includes characters from the query, security filters applied to the user, and all locations URL length. This means the query will get expanded against each of the keywords. For example, if a search query has 15 keywords and additional parameters and conditions, the query gets expanded 15 times, each with the other parameters and conditions in the query. So even though the number of characters in the search query may be below the limit, it's the expanded query that may contribute to exceeding this limit.  <br/> |**Mailboxes:** 10,000  <br/> **Sites:** 4,000 when searching all sites or 2,000 when searching up to 20 sites <sup>2</sup> <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @shelri please commit my suggestions if you agree.

**Comment by yogkumgit** [comment]
Gentle reminder @shelri Please update with feedback

**Comment by markjjo** [comment]
@yogkumgit I am working offline with @shelri and the PM on this PR. It's assigned to me and I will take ownership of any additional changes that need to be made.

**Comment by markjjo** [comment]
@shelri PR is merged. Apologies for the delay for confirmation from PG.

**Pull request merged**